railties/lib/rails/commands/routes/routes_command.rb

```diff
@@ -7,12 +7,14 @@ module Command
     class RoutesCommand < Base # :nodoc:
       class_option :controller, aliases: "-c", type: :string, desc: "Specifies the controller."
       class_option :grep_pattern, aliases: "-g", type: :string, desc: "Specifies grep pattern."
+      class_option :expanded_format, aliases: "--expanded", type: :string, desc: "Specifies grep pattern."
```

Comment:
I believe that `desc` should be rewrited.

Reply:
ouppps correct